app/views/checks/end_of_journey.html.slim

```diff
@@ -1,3 +1,4 @@
+- under_18_and_does_not_need_means_test = @check.under_eighteen_controlled_clr? || @check.under_eighteen_no_means_test_required?
```

Comment:
This is a bit redundant - ` Steps::Logic.under_eighteen_no_means_test_required?(session_data)` already returns true if ` Steps::Logic.controlled_clr?(session_data)` returns true (see logic.rb line 34), so you don't need to re-check that scenario here.

Reply:
Yeah, that makes sense. I was focussed on writing it, so I didn't miss both scenarios that I missed that. Will change now.